instrumentation/opentelemetry-instrumentation-flask/tests/test_programmatic.py

```diff
@@ -104,8 +106,8 @@ def expected_attributes_new(override_attributes):
     "http.server.duration": _server_duration_attrs_old_copy,
 }
 _recommended_metrics_attrs_new = {
-    "http.server.active_requests": _server_active_requests_count_attrs_new,
-    "http.server.request.duration": _server_duration_attrs_new_copy,
+    "http.server.active_requests": _server_active_requests_count_attrs_new_with_server_attributes,
+    "http.server.request.duration": _server_duration_attrs_new_with_server_attributes,
```

Comment:
Are we missing reflecting this change for the tests needed for `both_semconv`?

Reply:
I don't understand

Reply:
ok, I reverted this change after applying the suggestion below